assets/js/modules/adsense/util/is-zero-report.test.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tests for report utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies.
+ */
+import { isZeroReport } from './is-zero-report';
+
+describe( 'isZeroReport', () => {
+	it.each( [
+		[ 'undefined', undefined ],
```

Comment:
As mentioned, I think this one should return `undefined`.

Reply:
Updated.